### PR TITLE
Unit tests radio, saymode, and language prefix keys

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
This PR unit tests radio/saymode prefixes together, and language prefixes on their own, for any possible overlaps and duplicates

Moved mafia saymode key to :1 and holopad key to :2, we're running out of radio keys and i'm too afraid to move to 2-letter

## Why It's Good For The Game
1. Unit tests are good
2. https://github.com/tgstation/tgstation/pull/71326 this is bad

🆑 
spellcheck: Mafia changeling say prefix is now :1, and holopad say is now :2
/🆑

### CODE REVIEW

**Comment by etherware-novice** [comment]
damn i messed up hard this time

File: code/modules/unit_tests/key_prefixes.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+	// Now radioes
+
+	// Assoc list of key:name (or type in the cases of saymodes)
+	var/list/radio_keys = list()
+
+	for(var/radio_key in GLOB.department_radio_keys)
+		if(radio_keys.Find(radio_key))
+			TEST_FAIL("[GLOB.department_radio_keys[radio_key]] radio channel has the same prefix key ([radio_key]) as [GLOB.department_radio_keys[radio_keys.Find(radio_key)]]")
+		radio_keys[radio_key] = GLOB.department_radio_keys[radio_Key]
```
Comment (ZephyrTFA) [inline code/modules/unit_tests/key_prefixes.dm:21]:
```suggestion
		radio_keys[radio_key] = GLOB.department_radio_keys[radio_key]
```

File: code/modules/unit_tests/_unit_tests.dm [hunk 2]
```diff
@@ -118,6 +118,7 @@
 #include "hydroponics_validate_genes.dm"
 #include "inhands.dm"
 #include "keybinding_init.dm"
+#include "key_prefixes.dm"
```
Comment (Mothblocks) [inline code/modules/unit_tests/_unit_tests.dm:121]:
```suggestion
#include "key_prefixes.dm"
#include "keybinding_init.dm"
```

This is how the sort lines extension sorts it in VSC

**Review by Mothblocks — APPROVED** [review]
You need to put your holopad and mafia changes in the changelog. Also I'm fairly certain there's tips or examine texts or something referencing holopad being on :h. What language is on h?

**Comment by Mothblocks** [comment]
I meant reject but whatevs

**Comment by Zonespace27** [comment]
> You need to put your holopad and mafia changes in the changelog. Also I'm fairly certain there's tips or examine texts or something referencing holopad being on :h. What language is on h?

:h is taken up by the department shorthand channel

**Comment by GuiltyNeko** [comment]
AIs are the only ones that need to use hologram radio, right? And they don't have a "department shorthand channel" that they need to access... right?

**Comment by Mothblocks** [comment]
To be fair holopad speak shouldn't be a radio prefix anyway

That's not your problem but it is stupid

**Pull request merged**